packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -501,7 +501,7 @@ export class CdkToolkit {
 
         for (const name of Object.keys(deployResult.outputs).sort()) {
           const value = deployResult.outputs[name];
-          print('%s.%s = %s', chalk.cyan(stack.id), chalk.cyan(name), chalk.underline(chalk.cyan(value)));
+          print(`${chalk.cyan(stack.id)}.${chalk.cyan(name)} = ${chalk.underline(chalk.cyan(value))}`);
```

Comment:
the changes in this file are all refactors, right? we're not changing the logged message at all?

Reply:
Yup exactly.